underactuated.html

```diff
@@ -611,10 +622,22 @@ <h1>Honda's ASIMO vs. passive dynamic walkers</h1>
 
     </example>
 
-    <p> The underactuated systems are not feedback linearizable.  Therefore,
-    unlike fully-actuated systems, the control designer has no choice but to
-    reason about the nonlinear dynamics of the plant in the control design.
-    This dramatically complicates feedback controller design. </p>
+    <p> Fully-actuated systems are feedback equivalent to $\ddot\bq = \bu$,
+    whereas <i>underactuated systems are not feedback equivalent to $\ddot\bq =
+    \bu$</i>.  Therefore, unlike fully-actuated systems, the control designer
+    has no choice but to reason about the more complex dynamics of the plant in
+    the control design. When these dynamics are nonlinear, this can dramatically
+    complicate feedback controller design.
+    </p>
+
+    <p>A related concept is <a
+    href="https://en.wikipedia.org/wiki/Feedback_linearization">feedback
+    linearization</a>.  The feedback-cancellation controller in the example
+    above is an example of feedback linearization -- using feedback to convert a
+    nonlinear system into a linear one.  Asking whether or not a system is
+    "feedback linearizable" is not the same as asking whether it is
+    underactuated; a linear system can be underactuated and some nonlinear
```

Comment:
Continuing the line above: "underactuated; a CONTROLLABLE linear system can be underactuated (WHILE BEING TRIVIALLY FEEDBACK LINEARIZABLE)." I'd consider removing the sentence "and some nonlinear systems are only feedback linearizable to underactuated linear systems." since it takes a little to be parsed and doesn't add much to the first? I don't know if these modifications are too heavy, but I think they are important points.